packages/theme/src/Theme.ts

```diff
@@ -160,10 +154,20 @@ export class Theme extends HTMLElement implements ThemeKindProvider {
             const kindFragments = Theme.themeFragmentsByKind.get(
                 kind
             ) as FragmentMap;
-            const { [kind]: name } = this;
-            const currentStyles = kindFragments.get(name);
-            if (currentStyles) {
-                acc.push(currentStyles.styles);
+            const addStyles = (
+                name: FragmentType,
```

Comment:
Shouldn't this be of type `FragmentName`, as it can have values other than `'color' | 'scale' | 'core' | 'app'`?